source/fundamentals/run-command.txt

```diff
@@ -0,0 +1,165 @@
+   
+   .. code-block:: javascript
+
+      const commandOptions = { readPreference: "primary" };
+      const result = await myDB.command(commandDoc, commandOptions);
+   
+   For more information on read preference options, see :manual:`Read
+   Preference </core/read-preference/>` in the Server manual.
+
+Response
+--------
+
+Each method returns a ``Document`` object or a cursor that contains
+the response from the database after the command has been executed. Each
+database command performs a different function, so the response content
+can vary across commands. However, every response contains documents
+with the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field
+     - Description
+
+   * - <command result>
+     - Provides fields specific to the database command. For example,
+       ``count`` returns the ``n`` field and ``explain`` returns the
+       ``queryPlanner`` field.
+
+   * - ``ok``
+     - Indicates whether the command has succeeded (``1``)
+       or failed (``0``).
+
+   * - ``operationTime``
+     - Indicates the logical time of the operation. MongoDB uses the
+       logical time to order operations.
+       
+       .. seealso::
+  
+          To learn more about logical time, see our :website:`blog post about
+          the Global Logical Clock </blog/post/transactions-background-part-4-the-global-logical-clock>`.
+
+   * - ``$clusterTime``
+     - Provides a document that returns the signed cluster time. Cluster time is a
+       logical time used for ordering of operations.
+
+       The document contains the following fields:
+
+       - ``clusterTime``, which is the timestamp of the highest known cluster time for the member.
+       - ``signature``, which is a document that contains the hash of the cluster time and the ID
+         of the key used to sign the cluster time.
+
+Example
+-------
+
+The following code shows how you can use the ``runCursorCommand()`` method to
+run the ``checkMetadataConsistency`` command on the ``testDB`` database
+and iterates through the results:
+
+.. literalinclude:: /code-snippets/crud/runCommand.js
+   :language: javascript
+   :dedent:
+   :start-after: start-runcommand
+   :end-before: end-runcommand
+
+Output
+~~~~~~
+
+The output contains the contents of the cursor object. The documents
+describe any metadata inconsistencies in the database:
+
+.. code-block:: javascript
+
+   {
+     type: ...,
+     description: ...,
+     details: {
+       namespace: ...,
+       info: ...
+     }
+   }
+   {
+     type: ...,
+     description: ...,
+     details: {
+       namespace: ...,
+       collectionUUID: ...,
+       maxKeyObj: ...,
+       ...
+     }
+   }
+
+.. note::
+
+   If you store the command response in a cursor, you see only the
+   command result documents when you access the contents of the cursor. You won't
+   see the ``ok``, ``operationTime``, and ``$clusterTime`` fields.
```

Comment:
Q: is there a way to see this info when using a cursor?

Reply:
I don't think so. I can try printing the cursor a few different ways and see what happens, though.

Reply:
after some testing it seems like you cant